src/protocolsupport/protocol/typeremapper/tile/TileEntitySkullRemapper.java

```diff
@@ -25,10 +26,11 @@ protected static byte getLegacyData(BlockData skull) {
 
 	protected void register(List<Entry<Consumer<NBTCompound>>> list, Material skull, int skulltype) {
 		for (BlockData blockdata : MaterialAPI.getBlockDataList(skull)) {
-			byte rotation = getLegacyData(blockdata);
+			float rotation = getLegacyData(blockdata);
```

Comment:
why float?

Reply:
PE Stores rotation as a float. I tried with byte for PE's rotation and the rotation was off. On PC, rotation still looks normal even when I do use float. Also, should I make it detect if it is PE and PC and use those individual tags for each? Or should I write both since PC doesn't use Rotation and PE doesn't use Rot

Reply:
This should be still a byte, and pe rotation should be rotation * 22.5F.